app/views/idv/phone/new.html.erb

```diff
@@ -60,16 +62,43 @@
         required: true,
         class: 'margin-bottom-4',
       ) %>
+
+  <h2><%= t('idv.titles.otp_delivery_method') %></h2>
+  <p><%= t('idv.messages.otp_delivery_method_description') %></p>
+
+  <div class="grid-row grid-gap-2">
+    <div class="grid-col-12 tablet:grid-col-6">
+      <%= f.radio_button(
+            :otp_delivery_preference,
+            :sms,
+            checked: true, # We want SMS to be default checked
```

Comment:
How would this work if there's a vendor outage for SMS, since it would be checked but also disabled. IIRC in other screens this will be something more like... ? (with a corresponding assignment for "Voice" as well to select by default in the scenario that SMS is disabled)

```suggestion
            checked: !VendorStatus.new.vendor_outage?(:sms), # We want SMS to be default checked
```